README.md

```diff
@@ -27,6 +27,70 @@ $ IPFS_RUST_EXEC=$(pwd)/rust.sh npm test
 $ cat /tmp/rust.log
 ```
 
+# Troubleshooting hangs
+
+If the tests hang because of some deadlock on the `ipfs-http` side the tests
+will print the final summary from `mocha` and then seem to wait forever. This
+is true at least for the hangs seen so far, but you could run into a different
+issues. Note that in addition to test timeouts the http client also has a
+timeout, which can keep the `npm test` alive for less than ten seconds after
+the summary has been printed.
+
+There is [`why-is-node-running`] which is a tool which can dump all the reasons
```

Comment:
Probably don't need this paragraph at all

Reply:
Removed